.github/workflows/rebuilder.py

```diff
@@ -121,30 +177,28 @@ def get_monthly_rebuild_packages(pkgs: set[str], copr_pkgs: list[dict]) -> set[s
     >>> c = {"name" : "c", "builds" : { "latest" : { "id" : 2 } , "latest_succeeded" : { "id" : 1 } } }
     >>> d = {"name" : "d", "builds" : { "latest" : { "id" : 2 } , "latest_succeeded" : { "id" : 2 } } }
     >>> pkgs = { "b", "c", "d"}
-    >>> copr_pkgs = [a, b, c, d]
+    >>> copr_pkgs = [CoprPkg(p) for p in [a, b, c, d]]
```

Comment:
I don't really like the more work of having to adjust test cased but it shows how awesome this doc tests really are. The documentation is code that MUST run.